models/src_amazon_ads.yml

```diff
@@ -163,7 +163,7 @@ sources:
           - name: state
             description: "{{ doc('state') }}"
 
-      - name: portfolio_history
+      - name: portfolio_history 
```

Comment:
If you are allowing customers to disable the portfolio models, you will want to add a config here to disable the source as well.

See [here](https://github.com/fivetran/dbt_apple_search_ads_source/blob/db9bb313686b4ea19e3c2b41e6df4cb62917862e/models/src_apple_search_ads.yml#L209-L210) for an example.

Reply:
done